data/json/npcs/classes.json

```diff
@@ -222,7 +222,8 @@
       { "skill": "pistol", "bonus": { "rng": [ 1, 3 ] } },
       { "skill": "throw", "bonus": { "rng": [ 0, 2 ] } },
       { "skill": "barter", "bonus": { "rng": [ 2, 4 ] } }
-    ]
+    ],
+    "traits": [ [ "FELINE_EARS", 100 ] ]
```

Comment:
There's already a "traits" value on line 210. I also think you can make it randomly choose from the furry ear mutations by creating a trait group.